samples/bluetooth/iso_connected_benchmark/src/main.c

```diff
@@ -482,14 +486,14 @@ static int parse_interval_arg(struct bt_iso_chan_io_qos *qos)
 	return (int)interval;
 }
 
-static int parse_latency_arg(struct bt_iso_chan_io_qos *qos)
+static int parse_latency_arg(void)
```

Comment:
I liked it better when these parse functions did not use global variables.
But then again, this file has global variables, so it may be as clear to use them directly here as to pass them from the function above.
So, no requirement from me to change it.

Reply:
They had pointers before as there were two `bt_iso_chan_io_qos` structs to be used, and that was the way to avoid duplicating code. 

I don't disagree with you at all, but I think that's outside the scope of this PR.